apps/ledger-live-desktop/src/newArch/features/ModularDrawer/hooks/__tests__/useModularDrawerVisibility.test.ts

```diff
@@ -0,0 +1,97 @@
+import { renderHook } from "tests/testSetup";
+import { useModularDrawerVisibility, Location } from "../useModularDrawerVisibility";
+import * as featureFlags from "@ledgerhq/live-common/featureFlags/index";
+
+describe("useModularDrawerVisibility", () => {
+  it("should return false if the feature flag is disabled", () => {
+    jest.spyOn(featureFlags, "useFeature").mockReturnValue({
+      params: {
+        [Location.ADD_ACCOUNT]: true,
+      },
+      enabled: false,
+    });
+
+    const { result } = renderHook(() => useModularDrawerVisibility());
```

Comment:
Instead of the jest mocks I think it would be a more meaningful test to use:
```ts
renderHook(() => useModularDrawerVisibility(location)
  initialState: {
    settings: {
      overriddenFeatureFlags: {
        lldModularDrawer: {
          enabled: false,
          params: {
            [Location.ADD_ACCOUNT]: true,
           }
           ...
```